docs/installation.rst

```diff
@@ -5,14 +5,16 @@
 Installation
 ============
 
-To install |project_name|, you need `Python 3`_ and `pip`_ installed. You
-can then either
-`download it directly
+To install Borg-Import, you need `Python 3` and `pip` installed. You
+can then either `download borg-import directly
 <https://github.com/borgbackup/borg-import/archive/master.zip>`_
-or clone it from github_.
```

Comment:
Using "github" as label for that link was a bad idea.

Maybe "our github homepage"?

Also try to use something from global.rst.inc here.

Reply:
I changed the global.rst.inc's .git link's "name" and ran `find . -type f | xargs sed -i 's/|git_url|/_repo/g'` to ensure that I didn't break anything in the repo.